src/shipping/ups.md

```diff
@@ -25,7 +25,7 @@ To offer this shipping method to your customers, you must first open an account
 
 1. For a UPS XML account (default), set **UPS Type** to `United Parcel Service XML`. Then do the following:
 
-   - Enter the following UPS credentials: **User ID**, **Access License Number**, and **Password**
+   - Enter the following UPS credentials: **User ID**, **Access Key** (must be 16 symbols), and **Password**
```

Comment:
Thank you for your contributions, @ZhovnirMaria !

Please note, that `Access License Number` is correct Magento label. Could you please make a small editorial fix in your addition:

```
    - Enter the following UPS credentials: **User ID**, **Access License Number** (UPS `Access Key`, should be 16 symbols value), and **Password**
```

Reply:
Hi, @atwixfirster 
I made some changes to the request, you can check them.
Thank you!